week-3/quote-generator/quotes.js

```diff
@@ -16,10 +16,18 @@
 // pickFromArray(['a','b','c','d'])     // maybe returns 'c'
 
 // You don't need to change this function
-function pickFromArray(choices) {
-  return choices[Math.floor(Math.random() * choices.length)];
+function pickFromArray() {
```

Comment:
Nothing critical, but pay attention please that the name of the function matches its content.
It would be clearer if pickFromArray would get an array as a parameter and would return an element from it.
Then you would have another function which would assign the element to the page.

Reply:
ok got it